src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -193,30 +199,22 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
           maybeOutboundFeeCryptoBaseUnit
         )
       )
-        return
+        return null
       try {
         const amountCryptoBaseUnit = toBaseUnit(withdraw.cryptoAmount, asset.precision)
 
-        // No need to fetch a quote for the exact amount being withdrawn nor to estimate gas if the amount is too low
-        if (maybeOutboundFeeCryptoBaseUnit.isErr()) return
+        if (maybeOutboundFeeCryptoBaseUnit.isErr()) return null
         const outboundFeeCryptoBaseUnit = maybeOutboundFeeCryptoBaseUnit.unwrap()
 
-        if (bn(amountCryptoBaseUnit).lte(outboundFeeCryptoBaseUnit)) return
-
-        const withdrawBps = getWithdrawBps({
-          withdrawAmountCryptoBaseUnit: amountCryptoBaseUnit,
-          stakedAmountCryptoBaseUnit: opportunityData?.stakedAmountCryptoBaseUnit,
-          rewardsAmountCryptoBaseUnit: opportunityData?.rewardsCryptoBaseUnit?.amounts[0] ?? '0',
-        })
+        if (bn(amountCryptoBaseUnit).lte(outboundFeeCryptoBaseUnit)) return Err('Amount too low')
 
-        const quote = await getThorchainSaversWithdrawQuote({ asset, accountId, bps: withdrawBps })
-        if (!quote) return
+        if (!quote) return Err('Error getting THORChain savers quote')
```

Comment:
we don't need this check

Reply:
Nice catch - d5d948d4a84a8bbb4aa13a9fb890dfb938934cdc